docs/sharing.md

```diff
@@ -229,6 +229,75 @@ Content-Type: application/json
 }
 ```
 
+### GET /sharings/:sharing-id
+
+Get the information about a sharing. This includes the content of the rules, the members, as well as the already shared documents for this sharing.
+
+### Request
+
+```http
+GET /sharings/ce8835a061d0ef68947afe69a0046722 HTTP/1.1
+Host: alice.example.net
+Content-Type: application/vnd.api+json
+```
+
+```json
+{
+  "data": {
+    "type": "io.cozy.sharings",
+    "id": "ce8835a061d0ef68947afe69a0046722",
+    "meta": {
+      "rev": "1-4859c6c755143adf0838d225c5e97882"
+    },
+    "attributes": {
+      "description": "sharing test",
+      "preview_path": "/preview-sharing",
+      "app_slug": "drive",
+      "owner": true,
+      "created_at": "2018-01-04T12:35:08Z",
+      "updated_at": "2018-01-04T13:45:43Z",
+      "members": [
+        {
+          "status": "owner",
+          "name": "Alice",
+          "email": "alice@example.net",
+          "instance": "alice.example.net"
+        },
+        {
+          "status": "ready",
+          "name": "Bob",
+          "email": "bob@example.net"
+        }
+      ],
+      "rules": [
+        {
+          "title": "Hawaii",
+          "doctype": "io.cozy.files",
+          "values": ["612acf1c-1d72-11e8-b043-ef239d3074dd"],
+          "add": "sync",
+          "update": "sync",
+          "remove": "sync"
+        }
+    ],
+    "shared_docs": [
```

Comment:
I'd be nice to use the `relationships` of JSON-API, cf http://jsonapi.org/format/#document-resource-object-relationships (you can look at the `/files` endpoints, they use relationships).